docs/metrics/work-in-progress.md

```diff
@@ -0,0 +1,20 @@
+# Work in progress metrics
+
+## Work in progress
+
+To provide you with an high-level view of the current work in progress of your teams, Pulse groups pull requests that are currently open by the following phases:
+
+-   **Open:** the pull request is open but the team hasn't interacted or reviewed it.
+-   **In review:** the pull request received at least one review but it isn't approved yet. Reviews include approvals, change requests, and inline pull request comments, but don't include conversation comments.
+-   **Approved:** the pull request received at least one approval.
+
+## Work in progress details
+
+The following metrics for each open pull request allow you to understand which work items are about to become or have become blockers and may need an action to move forward:
+
+-   **Time in progress:** number of days that the pull request is in progress.
+-   **Phase:** [current phase](#work-in-progress) of the pull request, can be one of **Open**, **In review**, or **Approved**.
```

Comment:
does this work with adding the id? 😮

Reply:
Yes, this is a link to the URL fragment for the section above.

But now that I tested this, I don't think it makes sense to include the link because the page is so short that the whole content is displayed at once:

![image](https://user-images.githubusercontent.com/60105800/119674449-1e3fb300-be34-11eb-92dd-58749de2c7d3.png)

```suggestion
-   **Phase:** current phase of the pull request, can be one of **Open**, **In review**, or **Approved**.
```

Reply:
it is a valid url path

Reply:
> But now that I tested this, I don't think it makes sense to include the link because the page is so short that the whole content is displayed at once:

Since people use different devices and setups I think the extra link is always better.